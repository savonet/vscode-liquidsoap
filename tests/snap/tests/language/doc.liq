a = ()

# My function
# @param ~x The parameter.
def a.b(~x)
  x
end

def f()
  ignore(a)
  test.pass()
end

test.check(f)
