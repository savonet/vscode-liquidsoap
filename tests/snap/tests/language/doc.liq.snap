>a = ()
#^^^^^^^ source.liquidsoap
>
># My function
#^^^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
># @param ~x The parameter.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
>def a.b(~x)
#^^^ source.liquidsoap keyword.control.liquidsoap
#   ^^^^^^^^^ source.liquidsoap
>  x
#^^^^ source.liquidsoap
>end
#^^^ source.liquidsoap keyword.control.liquidsoap
>
>def f()
#^^^ source.liquidsoap keyword.control.liquidsoap
#   ^^^^^ source.liquidsoap
>  ignore(a)
#^^^^^^^^^^^^ source.liquidsoap
>  test.pass()
#^^^^^^^^^^^^^^ source.liquidsoap
>end
#^^^ source.liquidsoap keyword.control.liquidsoap
>
>test.check(f)
#^^^^^^^^^^^^^^ source.liquidsoap
>